ast/policy.go

```diff
@@ -144,27 +144,19 @@ type (
 		Location *Location
 	}
 
-	// Annotations contains information extracted from metadata in comments
-	Annotations interface {
-		annotationMaker()
-
-		// NOTE(tsandall): these are temporary interfaces that are required to support copy operations.
-		// When we get rid of the rule pointers, these may not be needed.
-		copy(Node) Annotations
-		node() Node
-	}
-
-	// SchemaAnnotations contains information about schemas
-	SchemaAnnotations struct {
-		SchemaAnnotation []SchemaAnnotation `json:"schemaannotation"`
-		Scope            string             `json:"scope"`
-		Rule             *Rule              `json:"-"`
+	// Annotations represents metadata attached to other AST nodes such as rules.
+	Annotations struct {
+		Location *Location           `json:"-"`
+		Scope    string              `json:"scope"`
+		Schemas  []*SchemaAnnotation `json:"schemas,omitempty"`
+		node     Node                `json:"-"`
```

Comment:
[nit] Do we need the json tag? Aren't private fields excluded anyways? I might be wrong though.

Reply:
forgot to remove the tag when I made the field private.